### ISSUE TITLE
Optimization: ship installed multi-start and sampled-objective workflows on common batching

### ISSUE THREAD

**Issue by nkiyohara** [nkiyohara/eqiora#1023]
## Outcome

Make the new capability usable through short installed examples: independent candidate evaluation for multi-start design, and one shared design optimized over fixed uncertainty samples. The same compiled Eqiora program supplies physics in both.

## Required behavior

- Provide readable ordinary `.eqi`/standard-package models and named physical inputs. Python constructs evaluation requests, objective reductions and optimizer calls; it does not duplicate the governing equations or own a solver loop.
- Show ordinary Python native batching and JAX `vmap`/`jit`/first-order AD using the same admitted signature. Distinguish per-design gradients from a summed/mean shared-design gradient.
- Use explicit samples/IDs and state whether the example computes a finite-sample objective or an admitted expectation estimator. Resampling and common-random-number comparison must be explicit.
- Demonstrate indexed diagnostics for one invalid candidate and refuse success-only objective averaging. Describe fixed Plan/shape/device bounds and the distinction between semantics and measured performance.
- Freeze a small reproducible workload with independent objective/gradient checks and a known bounded optimum or descent witness. Do not promise convergence for arbitrary nonlinear objectives.

## Target user code

The following is a proposed usage shape; final identifiers must match the shipped API and the program's exact ordered input schema:

```python
solve_one = eqjax.bind(program)

def loss_one(design, uncertainty):
    point = jnp.concatenate((design, uncertainty))
    prediction = solve_one(point)
    return jnp.mean((prediction - target) ** 2)

loss_many = jax.vmap(loss_one, in_axes=(None, 0))

def objective(design):
    return jnp.mean(loss_many(design, uncertainty_samples))

value, gradient = jax.jit(jax.value_and_grad(objective))(design)
```

The samples are fixed explicit inputs here. This code does not establish a derivative of a design-dependent probability law or a stochastic trajectory adjoint.

## Acceptance

Run both workflows from the installed package with bounded resources and compare native/JAX values and gradients against independent analytic or already accepted pointwise references. Include duplicates, shared design and one failure case. Update quick starts and the capability matrix only to the actually admitted scope; no separate example-only batch executor or redundant verification registry.

## Integration and verification

Use the existing compiler, Plan/Run, accepted evaluation, derivative, provider and artifact owners. Add a public abstraction only for the invariant and real consumers named here; no universal executor/registry or physics-name dispatch. Migrate affected pre-1.0 clients, examples and evidence atomically and delete displaced paths; published releases remain immutable. Run the narrow repository-owned checks, and every semantically affected registered case. Independently derive any new scientific expectation/tolerance at its claim boundary; ordinary composition checks reuse accepted pointwise evidence. Update the capability matrix when behavior lands, with contract, execution, verification and maturity stated separately. Native GitHub parent and blocking relations own the implementation order.
